message.go

```diff
@@ -184,3 +187,52 @@ func getServicesNotRunning(servicesRunningStatus map[string]bool) []string {
 	}
 	return servicesNotRunnig
 }
+
+func waitForResourcesRunning(previewName string) error {
+	retries := 0
+	wait := false
+	for retries < 5 {
+		resourceStatus, err := getResourceStatus(previewName)
+		if err != nil {
+			return err
+		}
+		for _, status := range resourceStatus {
+			if status != "running" {
```

Comment:
if `status` is `error` you can fast fail

Reply:
what happens if it fails because it does not find the database url and after a retry of the deployment it starts to work?